crowbar_framework/app/models/backup.rb

```diff
@@ -159,14 +159,19 @@ def save_archive
       return false
     end
 
+    if data.join("meta.yml").exist?
+      meta = YAML.load_file(data.join("meta.yml"))
```

Comment:
> to some point we have to rely on the user to just take a backup that was created by crowbar, and not put together by hand...

Sorry, replied too fast :-) I agree that we can't validate everything (like the content of the chef bits, for instance -- although it might actually be validated already?), but we do need at the very minimum some basic validation. 

TBH, I'm thinking we should validate more than what we do and check that the backup has all the required files, for instance -- to be sure that a restore won't fail because of a missing file.